modules/planning/common/open_space_info.h

```diff
@@ -292,7 +292,17 @@ class OpenSpaceInfo {
     return &debug_instance_;
   }
 
-  void sync_debug_instance() { debug_instance_.MergeFrom(*debug_); }
+  void sync_debug_instance() {
+    // Remove existing obstacle vectors to prevent repeating obstacle
+    // vectors.
+    if (debug_->planning_data().open_space().obstacles().size()) {
```

Comment:
Use `!debug_->planning_data().open_space().obstacles().empty()`.

Reply:
Fixed. Thank you.